Sources/FluidAudio/Diarizer/DiarizerTypes.swift

```diff
@@ -0,0 +1,204 @@
+import Foundation
+
+public struct DiarizerConfig: Sendable {
+    public var clusteringThreshold: Float = 0.7
+    public var minDurationOn: Float = 1.0
+    public var minDurationOff: Float = 0.5
+    public var numClusters: Int = -1
+    public var minActivityThreshold: Float = 10.0
+    public var debugMode: Bool = false
+
+    public static let `default` = DiarizerConfig()
+    
+    #if os(iOS)
+    public static let iosOptimized = DiarizerConfig(
+        clusteringThreshold: 0.7,
+        minDurationOn: 1.0,
+        minDurationOff: 0.5,
+        numClusters: -1,
+        minActivityThreshold: 10.0,
+        debugMode: false
+    )
+    #endif
+
```

Comment:
I don't see `iosOptimized` used, i think you meant to include it in the if for macos too?